crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -211,11 +212,32 @@ fn fetch_default_key(
 }
 
 #[get("")]
```

Comment:
should we use the same route convention as contexts i.e adding /list and support single get ?

Reply:
Single get is not required as of now,
we can develop it once needed
And /list makes sense, will make the change.